### PR TITLE
Aw/additional careplan changes 184988751

### PR DESCRIPTION
**Pull request by awisema**

### CODE REVIEW

File: db/reporting_structure.sql
Comment (eanders) [inline db/reporting_structure.sql:1]: Pretty sure this file shouldn't be changing

File: db/structure.sql
Comment (eanders) [inline db/structure.sql:1]: There should be no changes here either

File: db/warehouse_structure.sql
Comment (eanders) [inline db/warehouse_structure.sql:1]: There should be no changes here either

File: db/reporting_structure.sql
Comment (awisema) [reply db/reporting_structure.sql:1]: Oops -- I pulled back the last version of these files.

**Review by eanders — APPROVED** [review]

**Pull request merged**